resources/authorization/cmk-serviceaccount.yaml

```diff
@@ -2,3 +2,4 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: cmk-serviceaccount
+  namespace: cmk        
```

Comment:
Namespace added to service account yaml for testing - should be removed in upstream or replaced with comment
#namespace: user-supplied-namespace